libmscore/box.cpp

```diff
@@ -443,6 +443,18 @@ HBox::HBox(Score* score)
       setBoxWidth(Spatium(5.0));
       }
 
+//---------------------------------------------------------
+//   tick
+//---------------------------------------------------------
+
+Fraction HBox::tick() const
+      {
+      if (parent() && parent()->isMeasureBase())
+            return parent()->tick();
+      else
+            return MeasureBase::tick();
```

Comment:
If this is intended to be an override of `Element::tick()` function (as discussed [above](https://github.com/musescore/MuseScore/pull/6339#discussion_r455359445)) then its is better to call `Box::tick()` here as `Box` is the most direct base class of `HBox`. Right now it won't change anything but it will be better in case `Box` eventually gets its own `tick()` implementation.

Reply:
Very good point.  I miss a clean way to say "pass the call up to parent"